test/mappers/dissolution/dissolutionCreateRequest.mapper.test.ts

```diff
@@ -0,0 +1,27 @@
+import { assert } from 'chai'
+
+import DissolutionRequestMapper from 'app/mappers/dissolution/dissolutionRequest.mapper'
+import { DissolutionCreateRequest } from 'app/models/dto/dissolutionCreateRequest'
+import DirectorToSign from 'app/models/session/directorToSign.model'
+
+import { generateDirectorsToSign } from 'test/fixtures/session.fixtures'
+
+
+describe('Dissolution Request Mapper', () => {
+
+  const mapper: DissolutionRequestMapper = new DissolutionRequestMapper()
+
+  describe('mapToDissolutionRequest', () => {
+    it('should map director array to dissolution request', () => {
+      const directors: DirectorToSign[] = generateDirectorsToSign()
+      const result: DissolutionCreateRequest = mapper.mapToDissolutionRequest(directors)
+
+      result.directors.forEach((res, index) => {
```

Comment:
hmmm I'm not a fan of this test for two reasons:
1. If 0 directors are mapped, then this test will pass without any assertions being executed. This is why it is important to use zero logic in tests
2. We are relying on fixture data to support our assertions. We shouldn't. Fixtures are generic, valid objects and should be seen as starting points. Each unit test should use a fixture and overwrite it within a unit test to meet the test criteria, for example:
```
const director1: DirectorToSign = generateDirectorToSign()
director1.name = 'Director 1 name'

const director2: DirectorToSign = generateDirectorToSign()
director2.name = 'Director 2 name'

const result: DissolutionCreateRequest = mapper.mapToDissolutionRequest([director1, director2])

assert.equal(result.length, 2)
assert.equal(result[0].name, 'Director 1 name')
assert.equal(result[1].name, 'Director 2 name')
```